test/tests_defs.go

```diff
@@ -8,9 +8,10 @@ package test
 import "fmt"
 
 type testCase struct {
-	testName    string
-	args        *command
-	expectedErr string
+	testName       string
+	args           *command
+	expectedErr    string
+	blockedWarning *string
```

Comment:
```suggestion
	expectedWarning *string
```
Let's not assume all future warnings will be around blocked resources